airbyte-cdk/java/airbyte-cdk/datastore-postgres/src/test/kotlin/io/airbyte/cdk/db/PostgresUtilsTest.kt

```diff
@@ -108,12 +108,11 @@ internal class PostgresUtilsTest {
     }
 
     companion object {
-        lateinit var PSQL_DB: PostgreSQLContainer<Nothing>
+        var PSQL_DB: PostgreSQLContainer<Nothing> = PostgreSQLContainer<Nothing>("postgres:13-alpine")
```

Comment:
FYI you could even call the `start()` method using `run` or `apply` or whichever scoped method is best and get rid of `init()` entirely.